tools/normalize_mri_protocol_range_data.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * This script is the part of the second step for normalizing the mri_protocol table
+ * and mri_protocol_checks table.
+ * It populates the new min & max columns for each field that can contain ranges and
+ * returns SQL statements to remove the old %_range columns if the option "tosql" is set.
+ * This script must be run AFTER running this patch:
+ * /var/www/loris/SQL/New_patches/2018-02-27_normalize_mri_protocol_sql.
+ *
+ * "Usage: php normalize_mri_protocol_range_data.php [tosql]";
+ * "Example: php normalize_mri_protocol_range_data.php tosql";
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Liza Levitis <llevitis.mcin@gmail.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris
+ */
+require_once 'generic_includes.php';
+require_once 'Database.class.inc';
+require_once 'Utility.class.inc';
+
+// set default arguments
+$printToSQL = false;
+// SQL output
+$output     = "";
+
+// check if the 'tosql' option is passed
+if (!empty($argv[1]) && $argv[1] == "tosql") {
+  $printToSQL = true;
+}
+
+function split_ranges($table_name, $printToSQL, $output) {
+    $DB = \Database::singleton();
+
+// get all "ID" values from the mri_protocol table
+    $mp_idx = $DB->pselect("SELECT ID FROM $table_name", array());
+    $idx = array();
```

Comment:
@llevitis Normalization for 'mri_violations_log' table will not work as column 'ID' is not in mri_violations_log table.

Reply:
@sruthymathew123 I'm not normalizing `mri_violations_log` anymore - only `mri_protocol` and `mri_protocol_checks`.

Reply:
@llevitis In that case; ok.